src/Adapters/MockAdapter.ts

```diff
@@ -1005,4 +1018,27 @@ export default class MockAdapter
             });
         }
     }
+
+    async getTimeSeriesConnectionInformation(_adtInstanceIdentifier: {
+        id?: string;
+        hostName?: string;
+    }) {
+        try {
+            await this.mockNetwork();
+
+            return new AdapterResult({
+                result: new ADTInstanceTimeSeriesConnectionData({
+                    kustoClusterUrl: '',
```

Comment:
can we return some actual strings just so we can see them flowing through the system when we show them in the UI or write them to the config?